DESCRIPTION

```diff
@@ -0,0 +1,17 @@
+Package: exetera
+Type: Package
+Title: What the Package Does (Title Case)
```

Comment:
Should fill in these fields with actual values, so title is "Reproducible analysis pipelines for large tabular datasets" for example, other fields can be taken from ExeTera repo